src/modules/network/components/map/index.js

```diff
@@ -52,7 +52,7 @@ const getAttributionLinks = () => {
 
 const getTiles = () =>
   L.tileLayer(
-    'https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=pk.eyJ1IjoibWFudWxpZ2h0Y3VydmUiLCJhIjoiY2xodWVjYW52MGM0cDNobzYzZGRkOGU0dyJ9.aOMEZq0mbD6C-DgwU_ib_A',
+    'https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=pk.eyJ1IjoibWFudWxpZ2h0Y3VydmUiLCJhIjoiY2x1Nm8ybTVmMG9zMDJqbnZzNWoyZzc0bSJ9.4upFBJwPjObUVkzNwChDEw',
```

Comment:
I think this needs to be at least an environment variable which is injected in the built time

Reply:
@shuse2 Had considered that but for the most part, the environment variables used on Desktop are mostly in the webpack files or they are used but have a fallback value.

Reply:
Fixed